doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -289,14 +281,20 @@ Architecture they must be implemented as per the following specifications:
 ## 4.9 Kubernetes workloads
 
 In order for the Kubernetes workloads to be conformant with the Reference
-Architecture they must be implemented as per the following specifications:
+Architecture they must implement the parameter values as per the following specifications:
 
-|Ref|Specification|Details|Requirement Trace|
+|Ref|Parameter Group|Details|Requirement Trace|
 |---|---|---|---|
-|`ra2.app.001`||||
-|`ra2.app.002`||||
-|`ra2.app.003`||||
-|`ra2.app.004`||||
+|`ra2.app.001`|[Root](https://github.com/opencontainers/runtime-spec/blob/master/config.md)|Specifies the container's root filesystem.|TBD|
+|`ra2.app.002`|[Mounts](https://github.com/opencontainers/runtime-spec/blob/master/config.md#mounts)|Specifies additional mounts beyond root|TBD|
+|`ra2.app.003`|[Process](https://github.com/opencontainers/runtime-spec/blob/master/config.md#process)|Specifies the container process|TBD|
+|`ra2.app.004`|[Hostname](https://github.com/opencontainers/runtime-spec/blob/master/config.md#hostname)|Specifies the container's hostname as seen by processes running inside the container|TBD|
+|`ra2.app.005`|[User](https://github.com/opencontainers/runtime-spec/blob/master/config.md#user)|User for the process is a platform-specific structure that allows specific control over which user the process runs as|TBD|
```

Comment:
Mention the parameter groups in the spec, as there will be other specifications not related to parameter groups to come in the next release.
```suggestion
|`ra2.app.001`|[Root](https://github.com/opencontainers/runtime-spec/blob/master/config.md) Parameter Group (OCI Spec)|Specifies the container's root filesystem.|TBD|
|`ra2.app.002`|[Mounts](https://github.com/opencontainers/runtime-spec/blob/master/config.md#mounts) Parameter Group (OCI Spec)|Specifies additional mounts beyond root|TBD|
|`ra2.app.003`|[Process](https://github.com/opencontainers/runtime-spec/blob/master/config.md#process) Parameter Group (OCI Spec)|Specifies the container process|TBD|
|`ra2.app.004`|[Hostname](https://github.com/opencontainers/runtime-spec/blob/master/config.md#hostname) Parameter Group (OCI Spec)|Specifies the container's hostname as seen by processes running inside the container|TBD|
|`ra2.app.005`|[User](https://github.com/opencontainers/runtime-spec/blob/master/config.md#user) Parameter Group (OCI Spec)|User for the process is a platform-specific structure that allows specific control over which user the process runs as|TBD|
```